client/styles/components/_preferences.scss

```diff
@@ -212,6 +212,7 @@
 .preference__preview-button {
   @include themify() {
     @extend %preference-option;
+    color: #ed225d;
```

Comment:
can you please pull this color from the theme map rather than putting a raw color value that doesn't change with the theme inline?

Reply:
Oh my bad, I have done the changes. :)